src/Symfony/Component/Console/Tests/Fixtures/Style/SymfonyStyle/command/command_17.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
```

Comment:
No namespaces were introduced for all `command_*.php` files so IMHO we should let it this way.